delta/config/config.py

```diff
@@ -69,7 +69,7 @@ def validate_positive(num: Union[int, float], _: str) -> Union[int, float]:
         If number is not positive.
     """
     if num <= 0:
-        raise ValueError('%d is not positive' % (num))
+        raise ValueError(f'{num} is not positive')
```

Comment:
Personally love f-strings and it's my understanding they're the preferred format to use from Python 3.6 onwards, but they were introduced in 3.6 so we need to make sure that won't be a problem with previous version compatibility.

@bcoltin , right now the setup.py says >=3.5. Is there any reason we couldn't bump this up to 3.6?

Reply:
yeah, 3.6 should be fine

Reply:
@AliMuhammadOfficial can you add the version requirement change to the setup.py file and add that to this pull request?

Reply:
@bcoltin  Ok thanks for your precious  reply I create a pull request after changing setup.py